packages/cardano-services/src/Handle/TypeOrmHandleProvider.ts

```diff
@@ -46,22 +37,58 @@ export class TypeOrmHandleProvider extends TypeormProvider implements HandleProv
 
       await queryRunner.release();
 
-      const mapEntity = (entity: PartialHandleEntity | undefined): HandleResolution | null => {
+      const mapEntity = async (entity: HandleEntity | undefined): Promise<HandleResolution | null> => {
         if (!entity) return null;
         if (!entity.cardanoAddress) {
           this.logger.warn(`${entity.handle} has no associated address, which could be the result of a double mint.`);
           return null;
         }
 
-        const { cardanoAddress, handle, hasDatum, policyId } = entity;
-
-        return { cardanoAddress, handle, hasDatum, policyId, resolvedAt };
+        const {
+          cardanoAddress,
+          handle,
+          hasDatum,
+          policyId,
+          defaultForPaymentCredential,
+          defaultForStakeCredential,
+          asset
+        } = entity;
+
+        const nftMetadataRepo = dataSource.getRepository(NftMetadataEntity);
+        const handleMetadataRepo = dataSource.getRepository(HandleMetadataEntity);
+        const [nftMetadataEntity, handleMetadataEntity] = await Promise.all([
+          nftMetadataRepo.findOne({
+            order: { id: 'DESC' },
+            select: { id: true, image: true },
```

Comment:
I can't find where the `id` is used.
Could it be it can be removed from the `select` argument?
The same for next query.

Reply:
Typeorm is complaining when you order by field without selecting it